src/components/diagrams/networkAreaDiagram/network-area-diagram-content.tsx

```diff
@@ -148,7 +158,7 @@ function NetworkAreaDiagramContent(props) {
     useLayoutEffect(() => {
         if (props.svg) {
             const diagramViewer = new NetworkAreaDiagramViewer(
-                svgRef.current,
+                svgRef.current!,
```

Comment:
Proposition => ` if(props.svg && svgRef.current) {...}` then remove the force operator

Reply:
fixed